src/main/java/spoon/support/compiler/jdt/ReferenceBuilder.java

```diff
@@ -84,7 +86,6 @@
 import spoon.support.Level;
 import spoon.support.reflect.CtExtendedModifier;
 
-
```

Comment:
I miss this newline :(. Fun aside, could you add this newline back to keep this PR minimal, even if the newline seems wrong.